site/en/blog/how-blockbench-uses-the-eyedropper-api/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'How the 3D model editor Blockbench uses the EyeDropper API to let users choose colors from everywhere'
+subhead: >
+  The EyeDropper API provides a mechanism for creating an eyedropper tool that lets users sample colors from their screens, including outside of the browser window. Learn how the 3D model editor Blockbench uses this API in their codebase.
+date: 2023-01-31
+# updated: 2023-01-31
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/6fIc5QdWayhjyXmIFEi6.png
+alt: Computer created with Blockbench.
+authors:
+  - thomassteiner
+  - jannispetersen
+tags:
+  - capabilities
+  - case-study
+---
+
+[Blockbench](https://www.blockbench.net/) is a free, modern model editor for low-poly models with pixel art textures. If you have ever played [Minecraft](https://www.minecraft.net/), chances are you have seen assets that were created with Blockbench, for example the goat, which was added to Minecraft as part of the Caves and Cliffs Update, Part 1. The GPL version 3 licensed Blockbench code is [open source](https://github.com/JannisX11/blockbench) and maintained by a friendly community of developers.
```

Comment:
I've been thinking about a way to rephrase this last sentence to make it describe Blockbench more accurately, since I am the only maintainer and the main developer, and the community is actively testing and providing feedback. Maybe the best solution is just to remove the last half sentence and turn it into "The GPL version 3 licensed Blockbench code is open source on GitHub."

Reply:
Will change accordingly.